launch/launch/action.py

```diff
@@ -98,11 +98,7 @@ def describe_conditional_sub_entities(self) -> List[Tuple[
     ]]:
         """Override describe_conditional_sub_entities from LaunchDescriptionEntity."""
         return [
-            (
-                'Conditionally included by {}'.format(self.describe()),
-                entity,
-            )
-            for entity in self.get_sub_entities()
+            ('Conditionally included by {}'.format(self.describe()), self.get_sub_entities())
```

Comment:
I think the reason that this was structured as a `[(str, entity), ...]` before is that each entity may have a different reason for being included. As you have it here, matches what I had documented before, and should also work, `[(str, [entity]), ...]`, also allowing for groups to have the same condition reason. But I just wanted to reiterate clearly what the change in structure here was about.